src/cdev/cli/__main__.py

```diff
@@ -193,9 +193,30 @@ def wrapped_caller(*args, **kwargs):
             {"dest": "args", "nargs": argparse.REMAINDER},
         ],
     },
+    {
```

Comment:
Lets add an additional parameter to disable the prompt for deploying changes. maybe call it `--disable-prompt` or something like that. It should default to `False` that way someone does not accidently run it on a wrong environment . 

You will need to change the `deploy` command to take in a parameter also that bypasses the prompt.